src/components/Download/DownloadPanel/DownloadPanel.scss

```diff
@@ -50,15 +80,69 @@
   }
 
   @media screen and (min-width: $bpS) {
-    max-width: 30rem;
+    width: 60%;
+    min-width: 30rem;
+    max-width: 37.5rem;
 
-    /*display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
+    &--minimized {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+
+      .download-panel {
+        &__title {
+          margin-bottom: 0;
+        }
+      }
+    }
+
+    &--maximized {
+      flex-direction: column;
+      justify-content: normal;
+      align-items: normal;
+    }
+  }
+}
+
+//transitions
```

Comment:
If you have to split your code with comments, you must split in other files and iclude-it